cmd/ccboc/cmd/commands.go

```diff
@@ -364,6 +367,97 @@ func request(method, endpoint string, buffer *bytes.Buffer) ([]byte, *errorRespo
 	return body, nil, nil
 }
 
+func getCalculationPhase() error {
+	app := tview.NewApplication()
+	table := tview.NewTable().
+		SetBorders(true)
+
+	for {
```

Comment:
We don't need to do this in an infinite loop. You can remove this.

Reply:
Why? I really think this is necessary. This way lets us refetch the data and etc. easily.

Reply:
The only reason you need the infinite loop here is that you are forcing the app to stop every 3 seconds in the below code.
You need to check how the library dynamically updates data inside the app instead.

Reply:
If the app doesn't support it (unlikely), you should spawn a goroutine that grabs the new data every N seconds, and send the data inside the app via channels.